package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "setup-dbs": "psql -f ./db/setup.sql",
     "seed": "node ./db/seeds/run-seed.js",
-    "test": "jest",
+    "test": "PGDATABASE=nc_news_test jest",
```

Comment:
This shouldn't be necessary as the .env files already contain this information

Reply:
Thanks!